single-degree.php

```diff
@@ -60,6 +60,25 @@
 				</div>
 			<?php endif; ?>
 			</div>
+			<?php echo main_site_degree_display_subplans( $post->ID ); ?>
+			<?php if ( isset( $post_meta['degree_online_url'] ) && ! empty( $post_meta['degree_online_url'] ) ) : ?>
+			<aside class="online-callout">
+				<a class="media-background-container d-block h-100 py-3 px-3 px-lg-4 hover-parent bg-inverse hover-text-black text-decoration-none" href="<?php echo $post_meta['degree_online_url']; ?>" target="_blank">
+					<img src="<?php echo $online_bg_img; ?>" alt="" class=" media-background object-fit-cover hover-child hover-child-scale-up" data-object-fit="cover">
+					<div class="media-background object-fit-cover bg-inverse-t-3 hover-child hover-child-hide fade" data-object-fit="cover"></div>
+					<div class="media-background object-fit-cover bg-primary-t-1 hover-child hover-child-show fade" data-object-fit="cover"></div>
+					<div class="row">
+						<div class="col-auto">
+							<span class="fa fa-info-circle fa-3x align-middle"></span>
+						</div>
+						<div class="col">
+							<p class="font-weight-black align-middle">Did you know this program is available online?</p>
```

Comment:
styling adjustment ideas: add `h5` class here in order to increase the font-size for that a smidge? We could remove the font-weight-black class with that addition